asllib/doc/ASLTypingReference.tex

```diff
@@ -9656,16 +12726,128 @@ \section{Symbolic Expressions \label{sec:symbolicexpressions}}
 }
 \end{mathpar}
 
+\section{TypingRule.Normalize \label{sec:TypingRule.Normalize}}
+\hypertarget{def-normalize}{}
+The function
+\[
+\normalize(\overname{\staticenvs}{\tenv} \aslsep \overname{\expr}{\ve}) \aslto \overname{\expr}{\newe} \cup \overname{\TTypeError}{\TypeErrorConfig}
+\]
+symbolically simplifies an expression $\ve$ in the static environment $\tenv$,
+yielding an expression $\newe$.
+The result is a typing error, if one is detected.
```

Comment:
elsewhere the following phrase is used:
"The result is a type error, if one is detected."
it would be good to have consistency

Reply:
fixed